web/themes/custom/digital_gov/templates/node/node--topics.html.twig

```diff
@@ -179,11 +178,13 @@
     {# Display remaining resources, events and news for topic collection #}
     {% if is_first_page %}
       <section class="grid-container-desktop usa-section">
```

Comment:
Let's move this container into the conditional for the results so there isn't a visible empty space when no results are  returned.